recipe/meta.yaml

```diff
@@ -46,13 +50,13 @@ test:
     - test_load_elf.c                                                                                            # [linux and x86_64]
   commands:
     - if not exist %LIBRARY_INC%\\cudnn.h exit 1                                                                 # [win]
-    - if not exist %LIBRARY_INC%\\cudnn_adv_train.h exit 1                                                       # [win]
+    - if not exist %LIBRARY_INC%\\cudnn_adv.h exit 1                                                             # [win]
     - if not exist %LIBRARY_BIN%\\cudnn64_{{ '.'.join(cudnn_version.split('.')[0:1]) }}.dll exit 1               # [win]
-    - if not exist %LIBRARY_BIN%\\cudnn_adv_train64_{{ '.'.join(cudnn_version.split('.')[0:1]) }}.dll exit 1     # [win]
+    - if not exist %LIBRARY_BIN%\\cudnn_adv64_{{ '.'.join(cudnn_version.split('.')[0:1]) }}.dll exit 1           # [win]
     - test -f ${PREFIX}/include/cudnn.h                                                                          # [linux and x86_64]
-    - test -f ${PREFIX}/include/cudnn_adv_train.h                                                                # [linux and x86_64]
+    - test -f ${PREFIX}/include/cudnn_adv.h                                                                      # [linux and x86_64]
     - test -f ${PREFIX}/lib/libcudnn.so                                                                          # [linux and x86_64]
-    - test -f ${PREFIX}/lib/libcudnn_adv_train.so                                                                # [linux and x86_64]
+    - test -f ${PREFIX}/lib/libcudnn_adv.so                                                                      # [linux and x86_64]
     - test -f ${PREFIX}/lib/libcudnn.so.{{ '.'.join(cudnn_version.split('.')[0:1]) }}                            # [linux and x86_64]
     - test -f ${PREFIX}/lib/libcudnn.so.{{ '.'.join(cudnn_version.split('.')[0:3]) }}                            # [linux and x86_64]
     - ${GCC} test_load_elf.c -std=c99 -Werror -ldl -o test_load_elf                                              # [linux and x86_64]
```

Comment:
Could you please update `license` as `LicenseRef-Proprietary`?
See https://peps.python.org/pep-0639/#license-terms and https://spdx.github.io/spdx-spec/v2-draft/SPDX-license-expressions/:
`a user defined license reference denoted by the LicenseRef-[idString]`